app/stac_api/sample_data/swissMapRaster200/items/smr200-200-1-2019.json

```diff
@@ -0,0 +1,97 @@
+{
+  "stac_version": "0.9.0",
+  "stac_extensions": [
+    "proj",
+    "eo",
+    "checksum",
+    "https://data.geo.admin.ch/stac/geoadmin-extension/1.0/schema.json"
+  ],
+  "id": "smr200-200-1-2019",
+  "properties": {
+    "datetime": "2019-01-01",

```

Comment:
same here should we not have a full datetime ?

Reply:
dito, will cleanup with the spec PR